website/docs/add-secure-apps/providers/saml/index.md

```diff
@@ -14,20 +15,41 @@ Default fields are exposed through auto-generated Property Mappings, which are p
 
-You can download the metadata through the Webinterface, this link might be handy if your software wants to download the metadata directly.
+You can download the metadata through the web interface. This link might be useful if your software needs to fetch the metadata directly.
 
 The metadata download link can also be copied with a button on the provider overview page.
 
 ## Name ID
 
-You can select a custom SAML Property Mapping after which the NameID field will be generated. If left default, the following checks are done:
+You can select a custom SAML Property Mapping to control how the NameID field is generated. When using the default configuration, authentik determines the NameID value based on the requested format:
 
-- When the request asks for `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`, the NameID will be set to the hashed user ID.
-- When the request asks for `urn:oasis:names:tc:SAML:2.0:nameid-format:X509SubjectName`, the NameID will be set to the user's `distinguishedName` attribute. This attribute is set by the LDAP source by default. If the attribute does not exist, it will fall back the persistent identifier.
-- When the request asks for `urn:oasis:names:tc:SAML:2.0:nameid-format:WindowsDomainQualifiedName`, the NameID will be set to the user's UPN. This is also set by the LDAP source, and also falls back to the persistent identifier.
-- When the request asks for `urn:oasis:names:tc:SAML:2.0:nameid-format:transient`, the NameID will be set based on the user's session ID.
-- When the request asks for `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`, the NameID will be set to the user's email address.
+| Value                                 | NameID Format                                                          | Fallback              |
+| ------------------------------------- | ---------------------------------------------------------------------- | --------------------- |
+| Hashed user ID                        | `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`                 | N/A                   |
+| User's `distinguishedName` attribute¹ | `urn:oasis:names:tc:SAML:2.0:nameid-format:X509SubjectName`            | Persistent identifier |
+| User Principal Name (UPN)¹            | `urn:oasis:names:tc:SAML:2.0:nameid-format:WindowsDomainQualifiedName` | Persistent identifier |
+| User's session ID                     | `urn:oasis:names:tc:SAML:2.0:nameid-format:transient`                  | N/A                   |
+| User's email address                  | `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`               | N/A                   |
 
-    :::warning
-    Keep in mind that with the default settings, users are free to change their email addresses. As such it is recommended to use `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`, as this cannot be changed.
-    :::
+¹ These attributes are typically set by LDAP sources.
```

Comment:
Let's use an asterisk instead of the super-script 1, for a couple of reasons. 1) we don't use that style anywhere else in docs, and 2), there are two entries that use it, so having two 1s doesn't work. (But having an asterisk on both entries does.)